Python/samples/quanser-qube/tests/test_multi_concept.py

```diff
@@ -0,0 +1,72 @@
+"""
+Pytest for confirming multi concept brain functions with learned selector,
+one learned concept, and one programmed concept. Assessment with multi-concept 
```

Comment:
what do you mean by "assessment with multi-concept is not available"? It should work fine to assess the selector concept, though testing with the exported brain seems perfectly fine too. Suggest just removing this sentence.

Reply:
I learned this on Friday. I misunderstood M5 was assessment for programmed concepts (not selectors). I can fix